app/Dockerfile

```diff
@@ -28,6 +28,8 @@ COPY .coveragerc /src
 
 WORKDIR /
 
+RUN apt -y install curl
+
```

Comment:
@enjeyw Was there a reason this was removed or just an accident?

Reply:
accident, but it should be at the start, otherwise it'll kill our caches. I'll fix